stripper/ze_doom3_v2.cfg

```diff
@@ -1560,4 +1548,34 @@ modify:
 		"wait" "0"
 		"OnStartTouch" "!activatorSetHealth00-1"
 	}
+}
+; NPC cleanup before round ending. Should help eliminate the chances of an empty glow.
+add:
+{
+	"classname" "logic_eventlistener"
+	"EventName" "cs_pre_restart"
+	"origin" "-855 8672 55"
+	"IsEnabled" "1"
+	"TeamNum" "-1"
+	"FetchEventData" "0"
+	"OnEventFired" "npc_physbox1&*Kill0-1"
+	"OnEventFired" "npc_thruster_forward1&*Kill0-1"
+	"OnEventFired" "npc_thruster_side1&*Kill0-1"
+	"OnEventFired" "npc_physbox&*Kill0-1"
+	"OnEventFired" "npc_thruster_forward&*Kill0-1"
+	"OnEventFired" "npc_thruster_side&*Kill0-1"
+}
```

Comment:
Are you sure this issue was being created at round changes? This is a known client side bug with glow in CS:GO and it usually happens when the glow entity leaves the players rendered visleaves. You've probably seen it happen to the leader glow before.

Reply:
I'm able to partially replicate the glow bug when I leave where the NPC spawns and end the round. I assumed it may be due to round changes because during the last session there was an NPC glow on a bridge at the same spot two rounds in a row. After adding the event listener I can't replicate the glow bug anymore. I don't expect this to solve it completely but at least some of the time.